es6-classes/example-class.js

```diff
@@ -0,0 +1,18 @@
+class ExampleClass {
+  constructor(className) {
+    this.className = className;
+  }
```

Comment:
You can leave out the constructor completely if it's not needed.

Reply:
Gotcha, the constructor method def has been removed